packages/components/src/font-size-picker/utils.js

```diff
@@ -91,16 +87,24 @@ function getSelectOptions( optionsArray, disableCustomFontSizes ) {
 	} ) );
 }
 
-function getToggleGroupOptions( optionsArray, optionsContainComplexCssValues ) {
+/**
+ * Build options for the toggle group options.
+ *
+ * @param {Array}    optionsArray An array of font size options.
+ * @param {string[]} labelAliases An array of alternative labels.
+ * @return {Array}   Remapped optionsArray.
+ */
+export function getToggleGroupOptions(
+	optionsArray,
+	labelAliases = FONT_SIZES_ALIASES
+) {
 	return optionsArray.map( ( { slug, size, name }, index ) => {
-		let label = optionsContainComplexCssValues
-			? FONT_SIZES_ALIASES[ index ]
-			: size;
-		if ( ! optionsContainComplexCssValues && typeof size === 'string' ) {
-			const [ numericValue ] = splitValueAndUnitFromSize( size );
-			label = numericValue;
-		}
-		return { key: slug, value: size, label, name };
+		return {
+			key: slug,
+			value: size,
+			label: labelAliases[ index ] || size,
```

Comment:
I guess there shouldn't be a code path here where `labelAliases[ index ]` wouldn't be defined..
```suggestion
			label: labelAliases[ index ],
```